sources/packages/forms/src/form-definitions/studentdisabilityappeal.json

```diff
@@ -187,7 +187,7 @@
             }
           ],
           "className": "alert alert-info fa fa-info-circle",
-          "content": "<strong> Info!</strong><br /> Please note you must have your disability status verified by StudentAid BC to receive disability funding. If your disability status has already been verified, no further action is required.\n<br/><br/>\nIf your disability status has not yet been verified by StudentAid BC, you must complete an <a class=\"formio-href\" href=\"https://studentaidbc.ca/sites/all/files/form-library/appendix_8.pdf\" target=\"_blank\" rel=\"noopener noreferrer\">\nAppendix 8 - Disability Programs Application</a> to verify your permanent, or persistent or prolonged disability, and have your disability approved.\n<br/><br/>\nStudents with a permanent disability, or a persistent or prolonged disability may qualify for other programs. Contact the Accessibility Services Coordinator or financial aid office at your school, or visit <a class=\"formio-href\" href=\"https://studentaidbc.ca/\"target=\"_blank\" rel=\"noopener noreferrer\">\nwww.StudentAidBC.ca</a>",
+          "content": "<strong> Info!</strong><br /> Please note you must have your disability status verified by StudentAid BC to receive disability funding. If your disability status has already been verified, no further action is required.\n<br/><br/>\nIf your disability status has not yet been verified by StudentAid BC, you must complete an <a class=\"formio-href\" href=\"https://studentaidbc.ca/sites/all/files/form-library/appendix_8-parttime.pdf\" target=\"_blank\" rel=\"noopener noreferrer\">\nAppendix 8 for Part-Time Students - Disability Programs Application</a> to verify your permanent, or persistent or prolonged disability, and have your disability approved.\n<br/><br/>\nStudents with a permanent disability, or a persistent or prolonged disability may qualify for other programs. Contact the Accessibility Services Coordinator or financial aid office at your school, or visit <a class=\"formio-href\" href=\"https://studentaidbc.ca/\"target=\"_blank\" rel=\"noopener noreferrer\">\nwww.StudentAidBC.ca</a>",
```

Comment:
I see that this change is not captured in the ACs of the ticket. Please have the business add an AC for the appeal part as well.